WalletWasabi.Fluent/Models/Wallets/WalletModel.cs

```diff
@@ -30,33 +31,42 @@ public WalletModel(Wallet wallet)
 		Loader = new WalletLoadWorkflow(Wallet);
 		Settings = new WalletSettingsModel(Wallet.KeyManager);
 
-		RelevantTransactionProcessed =
-			Observable.FromEventPattern<ProcessedResult?>(Wallet, nameof(Wallet.WalletRelevantTransactionProcessed))
-					  .ObserveOn(RxApp.MainThreadScheduler);
+		_coinjoin = new(() => new WalletCoinjoinModel(Wallet, Settings));
+
+		var relevantTransactionProcessed =
+			Observable.FromEventPattern<ProcessedResult?>(Wallet, nameof(Wallet.WalletRelevantTransactionProcessed)).ToSignal()
+					  .Merge(Observable.FromEventPattern(Wallet, nameof(Wallet.NewFiltersProcessed)).ToSignal())
```

Comment:
This feels too much. We don't need to update everything when a new filter is received.

Reply:
This is intended to eventually replace `UiTriggers` which does [include filters](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Fluent/Models/UiTriggers.cs#L28)

Reply:
Okay, I see why... in some cases (Coins, transactions) we need to update the confirmation number on a new filter. But updating the address list for example doesn't make sense.

Ok for now, but keep it in mind for a potential future refactoring, we should not rebuild the whole list on a new filter, just update the confirmation number.